render_block/django.py

```diff
@@ -95,7 +113,7 @@ def _render_template_block_nodelist(nodelist, block_name, context):
         # If the wanted block was found, return it.
         if isinstance(node, BlockNode):
             # No matter what, add this block to the rendering context.
-            context.render_context[BLOCK_CONTEXT_KEY].push(node.name, node)
+            context.render_context[BLOCK_CONTEXT_KEY].blocks[node.name].insert(0, node)
```

Comment:
I'm unsure I'm following this change -- it looks like we're attempting to insert the current node at the beginning instead of the end? What's this change for?